exchanges/bitmex/bitmex_test.go

```diff
@@ -209,8 +210,42 @@ func TestGetInsuranceFundHistory(t *testing.T) {
 
 func TestGetLeaderboard(t *testing.T) {
 	t.Parallel()
-	_, err := b.GetLeaderboard(t.Context(), LeaderboardGetParams{})
-	require.NoError(t, err)
+
+	b := new(Bitmex) //nolint:govet // Intentional shadow
+	err := testexch.Setup(b)
+	require.NoError(t, err, "Setup must not error")
+
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, http.MethodGet, r.Method, "Request method should match expected")
+		assert.Equal(t, "/api/v1/leaderboard", r.URL.Path, "Request path should match expected")
+		w.WriteHeader(http.StatusOK)
+		_, err = w.Write([]byte(`[
+			{
+				"isRealName": true,
+				"name": "Trader Joe",
+				"profit": 12345.67
+			},
+			{
+				"isRealName": false,
+				"name": "CryptoKing",
+				"profit": 9876.54
+			}
+		]`))
+		assert.NoError(t, err, "Writing response to handler should not error")
+	}))
+	defer server.Close()
+
+	err = b.API.Endpoints.SetRunning(exchange.RestSpot.String(), server.URL+"/api/v1")
+	require.NoError(t, err, "SetRunning must not error")
+
+	expectedLeaderboard := []Leaderboard{
+		{IsRealName: true, Name: "Trader Joe", Profit: 12345.67},
+		{Name: "CryptoKing", Profit: 9876.54},
+	}
+
+	result, err := b.GetLeaderboard(t.Context(), LeaderboardGetParams{})
+	require.NoError(t, err, "GetLeaderboard must not error")
+	assert.Equal(t, expectedLeaderboard, result, "GetLeaderboard result should match expected")
```

Comment:
🦍🌴  `match expected` isn't bad, but we've used `be correct` pretty consistently. FSVO _we_.
```suggestion
	assert.Equal(t, expectedLeaderboard, result, "GetLeaderboard result should be correct")
```